src/inspector/worker_inspector.h

```diff
@@ -9,6 +9,7 @@
 #include <string>
 #include <unordered_map>
 #include <unordered_set>
+#include <cstdint>
```

Comment:
Unless there's a reason not to, this should be above the `#include <memory>` line to keep the includes in alphabetical order.